docs/user-manual/framework/memory-management/buffer-pool.md

```diff
@@ -1,15 +1,15 @@
-# Dynamic Memory and Buffer Management
+# Buffer Pools with Svc.BufferManager
 
 In embedded systems, dynamic memory allocation (a.k.a heap allocation) is typically avoided to reduce the steady-state
 variability in a running system. Avoiding dynamic memory allocation also avoids the problem of what to do in the case of
 a failed allocation. However, sometimes dynamic allocation provides for a simpler or more efficient solution.
 
-Safe dynamic allocation is available using the buffer manager pattern in F´. In short, this pattern allows components to
+Safe dynamic allocation is available using the buffer pool pattern in F´, implemented through the [`Svc::BufferManager`](../../../../Svc/BufferManager/docs/sdd.md) component. In short, this pattern allows components to
 dynamically allocate memory through a port call to a component designed to manage memory for the system. There are three steps in this process:
 
-1. Call allocation port receiving an`Fw::Buffer`
-2. Use allocated in the `Fw::Buffer`
-3. Call deallocation port providing the `Fw::Buffer`
+1. Call allocation port receiving an `Fw::Buffer`
+2. Use the allocated `Fw::Buffer`
```

Comment:
Need to add a step: check validity of buffer.

This is key because buffers can be returned as invalid if the buffer pool has run dry.